### PR TITLE
Update shrinkwrap

### PR DESCRIPTION
**Pull request by redmunds**
@chrisbank @jaredadobe @dirkschulze I'm not sure who's the best person to review this.

### CODE REVIEW

File: npm-shrinkwrap.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "generator-assets",
-  "version": "2.5.0-dev",
+  "version": "2.3.0-dev",
```
Comment (redmunds) [inline npm-shrinkwrap.json]: The script resets this version number. Who knows what it should be?

Comment (jaredadobe) [inline npm-shrinkwrap.json]: It is 2.3 in the package.json. Did we change it to 2.5 at some point and it got overwritten?

Comment (redmunds) [inline npm-shrinkwrap.json]: I manually incremented it for 0.5 and again for 0.6 since it was incremented for 0.4.

Comment (jaredadobe) [inline npm-shrinkwrap.json]: Do we know what it was for the last release? We should be higher than that.

**Comment by redmunds** [comment]
Reviewed by @jaredadobe. Gave me the "feel free to merge that PR once we bump that version" in Slack. Merging.

**Pull request merged**